docs/tools/tcmu-runner.md

```diff
@@ -50,10 +50,10 @@ target="_blank">GitHub</a>.  The project <a href="https://github.com/open-iscsi/
 target="_blank">*README*</a> file provides detailed information on how to
 compile, install and execute *tcmu-runner* daemon.
 
-The control of *tcmu-runner* emulated devices is achieved using the *targetcli*
-utility available as a package with most distributions. For instance, on
+The *targetcli* utility, which is available as a package in most distributions,
+controls devices that have been emulated with *tcmu-runner*. For example, on
```

Comment:
that have been emulated -> that are emulated

Reply:
Repaired.